packages/desktop-client/src/components/modals/CreateEncryptionKey.tsx

```diff
@@ -142,7 +152,9 @@ export default function CreateEncryptionKey({
                 <Input
                   type={showPassword ? 'text' : 'password'}
                   style={{ width: 300 }}
-                  onChange={e => setPassword(e.target.value)}
+                  onChange={(e: ChangeEvent<HTMLInputElement>) =>
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏we should be able to automatically infer this type from the `InputProps`. If it's currently not happening: would you mind checking why? Maybe we're missing definition of `onChange`.

Once we solve it in `InputProps`, then we won't need to do this change anymore.